plugins/org.locationtech.udig.tool.select/src/org/locationtech/udig/tool/select/ArrowSelection.java

```diff
@@ -31,6 +31,8 @@
 /**
  * Selects and drags single features.
  * 
+ * @deprecated {@link ArrowSelectionWithPopup} should be used instead
```

Comment:
Thats great to keep it in the code base, SDK-Users may still use this action in their application. Thinking about moving selection tools into a separate bundle to make it possible for SDK-Users to choose preferred selection tools. The "new" bundle should provide its own preferences too.

What do you think about creation a bundle 
* **org.locationtech.udig.tool.select.point.simple** for old code
* **org.locationtech.udig.tool.select.point.featureselect** for your contribution.

IMHO its a valid requirement if users of uDig SDK can choose to create their prefered application. In addition I see few other selection bundles (bbox, AOI, etc) not everybody wants to include per default.

However, my suggestions is to create another pull request that only has changes for deletion tool (radius and confirmation dialog, etc). With this pull request we can move on with the selection tool changes.

Now I'm wondering a bit about AbstractModalTool changes. Only ArrowSelectionWithPopup uses `SELECTION_SEARCH_SIZE` Please help me to understand..